modules/ptp-configuring-linuxptp-services-as-boundary-clock-dual-nic.adoc

```diff
@@ -0,0 +1,361 @@
+      delayAsymmetry                 0
+      fault_reset_interval         4
+      neighborPropDelayThresh      20000000
+      masterOnly                     0
+      G.8275.portDS.localPriority  128
+      #
+      # Run time options
+      #
+      assume_two_step              0
+      logging_level                6
+      path_trace_enabled         0
+      follow_up_info               0
+      hybrid_e2e                   0
+      inhibit_multicast_service  0
+      net_sync_monitor           0
+      tc_spanning_tree           0
+      tx_timestamp_timeout       10
+      #was 1 (default !)
+      unicast_listen          0
+      unicast_master_table  0
+      unicast_req_duration  3600
+      use_syslog              1
+      verbose                   0
+      summary_interval      -4
+      kernel_leap             1
+      check_fup_sync          0
+      #
+      # Servo Options
+      #
+      pi_proportional_const     0.0
+      pi_integral_const         0.0
+      pi_proportional_scale     0.0
+      pi_proportional_exponent  -0.3
+      pi_proportional_norm_max  0.7
+      pi_integral_scale         0.0
+      pi_integral_exponent      0.4
+      pi_integral_norm_max      0.3
+      step_threshold              0
+      first_step_threshold      0.00002
+      max_frequency               900000000
+      clock_servo                 pi
+      sanity_freq_limit         200000000
+      ntpshm_segment              0
+      #
+      # Transport options
+      #
+      transportSpecific   0x0
+      ptp_dst_mac          01:1B:19:00:00:00
+      p2p_dst_mac          01:80:C2:00:00:0E
+      udp_ttl                1
+      udp6_scope           0x0E
+      uds_address          /var/run/ptp4l
+      #
+      # Default interface options
+      #
+      clock_type             BC
+      network_transport      L2
+      delay_mechanism        E2E
+      time_stamping          hardware
+      tsproc_mode            filter
+      delay_filter           moving_median
+      delay_filter_length  10
+      egressLatency          0
+      ingressLatency         0
+      boundary_clock_jbod  1
+      #
+      # Clock description
+      #
+      productDescription    ;;
+      revisionData            ;;
+      manufacturerIdentity  00:00:00
+      userDescription         ;
+      timeSource              0xA0
+    ptpSchedulingPolicy: SCHED_OTHER
+    ptpSchedulingPriority: 65
+  ptpClockThreshold:
+    holdOverTimeout: 5
+    maxOffsetThreshold: 100
+    minOffsetThreshold: -100
+  recommend:
+  - profile: "profile2"
+    priority: 10
+    match:
+    - nodeLabel: "node-role.kubernetes.io/worker"
+      nodeName: "compute-0.example.com"
+----
++
+[NOTE]
+====
+To disable the `phc2sys` service on the second NIC, you need to completely remove the `phc2sysOpts` field from the `PtpConfig` CR.
+====
+
+. Create the CR by running the following command:
++
+[source,terminal]
+----
+$ oc create -f boundary-clock-ptp-config-nic2.yaml
+----
+
+.Verification steps
```

Comment:
The proper heading per the mod docs templates is just `.Verification`

https://raw.githubusercontent.com/redhat-documentation/modular-docs/master/modular-docs-manual/files/TEMPLATE_PROCEDURE_doing-one-procedure.adoc

Reply:
Thanks! Updated in a few places.